### PR TITLE
Setup landing page

### LINKED ISSUE #3: Set up Landing Page
Use Material UI to develop landing page - https://www.notion.so/Designs-c4aafb2fc93240aea1b033110db076cd#7edcae8b13d34f3284e3a8474d283170

## Implementation Notes
It doesn't have to be pixel perfect, use Material UI components that are closest to what's on the design.

### PR DESCRIPTION
**Pull request by angelocordon**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Update Landing Page UI with registration/authentication form and copy.

## Related Tickets & Documents
Closes #3 

## Before & After
**Before**
image

**After**
image

image

### CODE REVIEW

**Comment by angelocordon** [comment]
@lpatmo - just a heads up, this may tie into your #65 PR; may look into how we can merge the two together. This is primarily just the UI work.

File: src/components/Home/AuthForms/AuthForm.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const AuthForm = () => {
+  const [activeSignUpForm, setActiveSignUpform] = useState(true);
+
+  const toggleActiveForm = event => {
+    event.preventDefault();
+    setActiveSignUpform(!activeSignUpForm);
+  };
+
+  return (
+    <Box component={Paper} elevetion={3} padding={3}>
```
Comment (lpatmo) [inline src/components/Home/AuthForms/AuthForm.js]: Is `elevetion` meant to be `elevation`?

File: src/components/Home/AuthForms/AuthForm.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      {activeSignUpForm && <SignUpForm toggleActiveForm={toggleActiveForm} />}
```
Comment (lpatmo) [inline src/components/Home/AuthForms/AuthForm.js]: Interesting approach -- I'm used to writing ```{activeSignUpForm ? <SignUpForm toggleActiveForm={toggleActiveForm} /> : <LoginForm>}```
, but this is fine :)

**Comment by angelocordon** [comment]
@lpatmo @sebbel - fixed up changed based on feedback - please merge if acceptable

**Pull request merged**